GameServerLib/Logic/GameObjects/Unit.cs

```diff
@@ -317,6 +400,11 @@ public override void onCollision(GameObject collider)
         /// </summary>
         public virtual void AutoAttackHit(Unit target)
         {
+            if (HasCrowdControl(CrowdControlType.Blind)) {
+                // a modified DealDamageTo() to set damage text to "blind"
```

Comment:
This can be done easily by:
1. Making current method have a `DamageText` value instead of `bool`.
2. Add an overload like this:
```
DealDamageTo(... , bool isCrit)
{
    if (isCrit) DealDamageTo(..., DamageText.Critical);
    else DealDamageTo(..., DamageText.Normal);
}
```
(Just a psuedo code)